model/plugins/delete_user/structures.smithy

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+
+@input
+structure DeleteUser_Input {
+    @required
+    @httpLabel
+    username: String
+}
+
+@output
+structure DeleteUser_Output {
+    response: Response
```

Comment:
Can we keep the labels consistent, you can call this `content:`?

Reply:
Yeap! I'll replace it in few minutes